repos/system_upgrade/el7toel8/actors/checkbind/libraries/model.py

```diff
@@ -0,0 +1,92 @@
+from leapp.models import BindFacts, BindConfigIssuesModel
+from leapp.libraries.common import isccfg
+from leapp import reporting
+
+
+def add_statement(statement, state):
+    """ Add searched statement to found issues """
+
+    stmt_text = statement.serialize_skip(' ')
+    name = statement.var(0).value()
+    if name in state:
+        state[name].append((stmt_text, statement.config.path))
+    else:
+        state[name] = [(stmt_text, statement.config.path)]
+
+
+def find_dnssec_lookaside(statement, state):
+    try:
+        assert statement.var(0).value() == 'dnssec-lookaside'
+
+        arg = statement.var(1)
+        if arg.type() == arg.TYPE_BARE and arg.value() in ['auto', 'yes']:
+            # auto or yes statement
+            add_statement(statement, state)
+        # dnssec-lookaside "." trust-anchor "dlv.isc.org";
+        elif (arg.type() == arg.TYPE_QSTRING and arg.value() == '"."'
+              and statement.var(2).value() == 'trust-anchor'
+              and statement.var(3).invalue() == 'dlv.isc.org'):
+            add_statement(statement, state)
```

Comment:
seems too big clause under the try. Consider use try-except-finally structure

Reply:
I don't get what I am supposed to do. For cases when keyword would have unexpected parameters, that means for example *dnssec-lookaside ".";* It would be invalid configuration for named, but I think it should not be fatal error. It might be permitted in future versions.

This code will add statement to state:
dnssec-lookaside yes;
dnssec-lookaside auto;
dnssec-lookaside "." trust-anchor "dlv.isc.org";

But it would ignore:
dnssec-lookaside no;

Another option might be removing try and let it crash hard, when unexpected configuration is found. But I don't know how to use finally in this case.

Reply:
Ok, resolved by just checking for dnssec-lookaside no; which is the only supported configuration forwad.